Shoko.Server/Renamer/RenameFileHelper.cs

```diff
@@ -20,8 +20,16 @@ namespace Shoko.Server
 {
     public class RenameFileHelper
     {
+        public class RenamerInfo
+        {
+            public int ID { get; set; }
+            public string Name { get; set; }
+            public string Description { get; set; }
+            public Type Type { get; set; }
+        }
```

Comment:
What is even the reason for changing how the renamer info is stored- to get extra data? But the only thing there is added is a false "ID" which is already defined by the IDictonary key.

Reply:
Removed.